src/scripts/main.js

```diff
@@ -1,3 +1,301 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+
+let isAscending = true;
+
+const sortTable = function(index, type) {
+  const compare = function(rowA, rowB) {
+    const rowADate = rowA.cells[index].innerHTML;
+    const rowBDate = rowB.cells[index].innerHTML;
+
+    switch (type) {
+      case 'Name':
+      case 'Position':
+      case 'Office':
+        return isAscending ? rowADate.localeCompare(rowBDate)
+        : rowBDate.localeCompare(rowADate);
+      case 'Age':
+        return isAscending ? rowADate - rowBDate : rowBDate - rowADate;
+      case 'Salary':
+        const dateA = rowADate.replace(/[$,]/g, '');
+        const dateB = rowBDate.replace(/[$,]/g, '');
+
+        return isAscending ? dateA - dateB : dateB - dateA;
```

Comment:
Not fixed, please add the default case

<img width="650" alt="Screenshot 2023-04-17 at 18 30 02" src="https://user-images.githubusercontent.com/32853034/232551122-bdcab240-9040-4dfa-8304-8d14783f0d6d.png">

```suggestion
        return isAscending ? dateA - dateB : dateB - dateA;
        
        default: 
        ...
        
```

Reply:
@agniya-i  I'm sorry, I don't fully understand what this means (